src/utils/misc.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Returns an array with unique values
+ *
+ * @param array The input array with possibly redundant values
+ */
+export function unique<T>(array: T[]): T[] {
+  return [...new Set(array)];
+}
```

Comment:
A general question: why don't we use `lodash` and `moment.js` in our project?

Reply:
I read multiple negative articles about using `lodash` including https://codeburst.io/why-you-shouldnt-use-lodash-anymore-and-use-pure-javascript-instead-c397df51a66.

I've only needed simple addition/subtraction to Date objects, and d3 handled the formatting part. I don't have much objection to using date/time libraries to improve readability for more complicated manipulation or formatting.

Also, you might be interested in using [`luxon`](https://moment.github.io/luxon/), which is derived from `moment.js`.